src/scss/components/optional/_listings.scss

```diff
@@ -64,6 +64,7 @@ $listing--inactive-bg           : rgba($listing-bg, 0.5);
 
         &.has-noItems {
             border: 0;
+            display: none;
```

Comment:
Probably don't need the border if it's `display: none`?

Reply:
good point!